spring-cloud-gcp-secretmanager/src/main/java/org/springframework/cloud/gcp/secretmanager/SecretManagerOperations.java

```diff
@@ -40,6 +40,21 @@
 	 */
 	void createSecret(String secretId, String payload);
 
+	/**
+	 * Creates a new secret or a new version of existing secret with the provided
+	 * {@code payload} for a specific {@code projectId}.
+	 *
+	 * <p>
+	 * If there is already a secret saved in SecretManager with the specified
+	 * {@code secretId}, then it simply creates a new version under the secret with the secret
+	 * {@code payload}.
+	 *
+	 * @param secretId the secret ID of the secret to create.
+	 * @param payload the secret payload string.
+	 * @param projectId unique identifier of your project.
+	 */
+	void createSecretWithProject(String secretId, String payload, String projectId);
```

Comment:
I would just create an overloaded method `createSecret(secretId, payload, projectId)`

Reply:
done in newer commit